.github/workflows/build-and-deploy-container-image.yaml

```diff
@@ -388,13 +388,23 @@ jobs:
       - check-code-compliance
 
     steps:
+      - name: find dockerfile
+        id: build-this
```

Comment:
Das können wir doch auch in `validate-event` einbauen und dann diesen step nach `prepare` umbenennen.